python/xrprimer/utils/synbody_utils.py

```diff
@@ -0,0 +1,102 @@
+"""Utils for load annotations from OpenXDLab SynBody dataset.
+
+Requirements:
+
+```
+pip install numpy imath openexr flow_vis
+```
+"""
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple, Union
+
+import flow_vis
+import Imath
+import OpenEXR
+import numpy as np
+
+PathLike = Union[str, Path]
+
+
+class ExrReader:
```

Comment:
As offline discussion, we may extract the general part of `ExrReader` to `io`, and write a `SynBodyReader` to read all necessary data from synbody.

Reply:
Done with `io/exr_reader.py`